e2e/images/README.md

```diff
@@ -1 +1,52 @@
-# rook-test-infrastructure
+# Rook Test Infrastructure
```

Comment:
why is this readme separate from the one in the parent e2e folder? seems like it would all be the parent folder